### PR TITLE
A6b (0.06) changes

### PR DESCRIPTION
**Pull request by OJCubing**
- Allows more Delegate discretion in giving extras for times above 0.06 seconds
- Clarifies "strictly less than 0.06" means "0.05 seconds or less"
- Moves a guideline to a more appropriate place

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -431,8 +431,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
-        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
-        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
+        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time of 0.05 seconds or lower, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
```
Comment (Jobarion) [inline wca-regulations.md]: This is not correct. A timer showing a time of `0.053` should result in an extra, but that's not covered by this sentence.

**Comment by CarterKoala** [comment]
Simplified this proposal to only the A6b2 change. The .05 change will not be implemented, and the Guideline renumbering will be done in a separate PR

File: wca-regulations.md [hunk 2]
```diff
@@ -432,7 +432,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6a) The competitor releases the puzzle before stopping the solve, and stops the solve by stopping the timer. If a stopwatch is in use, the judge stops the stopwatch as soon as the competitor does this.
     - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
         - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately or as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
```
I think were were also in favor of something like this to allow for more situations in which an extra would not be automatic?

File: wca-regulations.md [hunk 3]
```diff
@@ -431,8 +431,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
+        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately or as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
+        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: The WCA Delegate may provide an extra attempt if they believe the timer may have malfunctioned and they are sure the competitor did not stop the timer deliberately.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: The WCA Delegate may provide an extra attempt if they believe the timer may have malfunctioned and the competitor did not stop the timer deliberately.
```
"they are sure" is a high standard for evidence I don't think we use elsewhere. Whether something happened or not should be up to the Delegate to decide if there isn't conclusive evidence.

File: wca-regulations.md [hunk 4]
```diff
@@ -431,8 +431,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
-        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
-        - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
+        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately or as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. Exception: No extra attempt is required if the WCA Delegate determines that the timer was stopped deliberately, or stopped as a result of the competitor not starting the timer correctly (see [Regulation A4](regulations:regulation:A4)).
```

**Pull request merged**